dashboards-observability/public/components/integrations/plugins/app_list.ts

```diff
@@ -0,0 +1,32 @@
+export const apps = [
+  {

```

Comment:
You can use this section to manage everything related to the app. You can add flyout content directly or use a new MD file to have the content and add the path to the MD file here.